code/src/components/HappySingleThought.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { formatDistanceToNow } from 'date-fns'
+
+const HappySingleThought = ({ thoughtId, message, hearts, creationDate, handleHeartClick }) => {
+  const creationDateObject = new Date(creationDate)
+  const formattedCreationDate = formatDistanceToNow(creationDateObject, { addSuffix: true })
```

Comment:
This two lines, together with line 23!! Nice approach!
I used the following instead:
`<p className="thoughts-time">{formatDistance(new Date(thought.createdAt), new Date())} ago</p>`